go/engine/paperprovision.go

```diff
@@ -222,11 +222,14 @@ func (e *PaperProvisionEngine) makeDeviceWrapArgs(m libkb.MetaContext) (*DeviceW
 	}, nil
 }
 
-// copied from loginProvision
-// makeDeviceKeys uses DeviceWrap to generate device keys.
+// copied from loginProvision makeDeviceKeys uses DeviceWrap to
```

Comment:
nit: this should be two separate lines or put a period between them since it's two separate thoughts.